libvast/src/command.cpp

```diff
@@ -196,6 +197,16 @@ void render_parse_error(const command& cmd,
   }
 }
 
+void manualtext(const command& cmd, std::ostream& os,
```

Comment:
I find this hard on they eyes when reading. We have `helptext`, which barely works, and I'd suggest `mantext` to keep it simple. Otherwise I'd suggest using `_` for clearer function naming.

Reply:
Also, could you a small documentation string that tells the user that we are generating Markdown here?